docs/how-to/build-models-into-containers.md

```diff
@@ -0,0 +1,43 @@
+# Build models into containers
+
+This guide assumes you are in the `kubeai.git` repo. In this guide we will preload a LLM into a custom built Ollama serving image. You can follow the same steps for other models and other serving engines.
+
+Define some values
+```bash
+export MODEL_URL=ollama://qwen2:0.5b
+
+# Customize with your own image repo.
+export IMAGE=us-central1-docker.pkg.dev/substratus-dev/default/ollama-builtin-qwen2-05b:latest
+```
+
+Build and push image. Note: building (downloading base image & model) and pushing (uploading image & model) can take a while depending on the size of the model.
+
+```bash
+# Within kubeai.git...
```

Comment:
I would probably just add a `git clone` command and then remove "This guide assumes you are in the `kubeai.git` repo."

Reply:
UX is better if you can just copy paste a command instead of figuring out `kubeai.git` where is that hmm

Reply:
Done